src/extension.ts

```diff
@@ -64,7 +65,11 @@ export function activate(context: vscode.ExtensionContext) {
         }));
     context.subscriptions.push(vscode.commands.registerCommand(
         `${SETTINGS_VIEW_NAME}.refresh`,
-        () => cdpTargetsProvider.refresh()));
+        async () => {
+            cdpTargetsProvider.refresh();
+            await cdpTargetsProvider.getChildren();
+            vscode.window.showInformationMessage("Refresh complete");

```

Comment:
It would be nice if it would say the number of targets found as well.  Maybe a message such as "Refresh complete.  {num} targets found"

Reply:
Good idea, I'll add that!